# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sbc/hotfix-x86.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sbc-1.5/sbc/sbc_primitives.c.vanilla	2021-10-10 17:27:30.843837553 +0200
+++ sbc-1.5/sbc/sbc_primitives.c	2021-10-10 17:36:16.652806213 +0200
@@ -593,7 +593,7 @@
 
 static void sbc_init_primitives_x86(struct sbc_encoder_state *state)
 {
-#if SBC_BUILD_WITH_MMX_SUPPORT == 1 || SBC_BUILD_WITH_MMX_SUPPORT == 1
+#if defined(SBC_BUILD_WITH_MMX_SUPPORT) || defined(SBC_BUILD_WITH_MMX_SUPPORT)
 	__builtin_cpu_init();
 #endif
 
